### ISSUE TITLE
Does not compile on Windows

### ISSUE THREAD

**Issue by levrik** [baliance/gooxml#160]
## Description

Trying to use `document.New()` on Windows.

## Expected Behavior

It compiles.

## Actual Behavior

Getting an error:
```
go build baliance.com/gooxml/schema/soo/wml: C:\Go\pkg\tool\windows_amd64\compile.exe: fork/exec C:\Go\pkg\tool\windows_amd64\compile.exe: The filename or extension is too long.
```

**Comment by levrik** [comment]
This seems to be related to the max path length on Windows. Sadly I am not able to move my Go workspace to another location that is higher in the tree. Does the packages of this library need to be that deeply nested?

**Comment by levrik** [comment]
After reading through the whole issue I think this issue does not primarily come from the location of my workspace. I think it's related to the amount of files that are inside that folder. This exceeds the maximum length of the whole command because (if I understood that correct) the Go compiler just appends all files per folder (or package?) to that one.
Looks like this should be resolved on the Go-side. But not sure what can I do now to get this library working 😕

**Comment by tbaliance** [comment]
What version of Windows?  The longest path appears to be:

```
baliance.com/gooxml/schema/soo/dml/spreadsheetDrawing/CT_GraphicalObjectFrameNonVisual_test.go

``` 
Which his only 94 characters.  I've tried to flatten it before, but it's not easy.  The schema code is generated and there are several types that have the same name.

**Comment by daniel-007** [comment]
I have the same problem,my env:
windows10 64bit 
go version go1.10 windows/amd64

error info:
baliance.com/gooxml/schema/soo/sml
go build baliance.com/gooxml/schema/soo/sml: D:\devtools\Go\pkg\tool\windows_amd64\compile.exe: fork/exec D:\devtools\Go\pkg\tool\windows_amd64\compile.exe: The filename or extension is too long.

**Comment by levrik** [comment]
@tbaliance I was able to move my Go workspace up. That resolved the issue for me. I think we have to wait for the Go developers resolve it.

**Comment by freb** [comment]
Not sure if this will help, but Windows 10 supports longer paths, you jsut have to enable it in the registry: <https://superuser.com/questions/1119883/windows-10-enable-ntfs-long-paths-policy-option-missing>

**Comment by tbaliance** [comment]
This should be fixed in Go 1.10.3 when released in mid June and will be fixed in Go 1.11 when it's released in August.

**Comment by tbaliance** [comment]
This should be resolved now, please let me know if you run into any more issues.
